microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -400,6 +420,33 @@ Kids Mode does not collect data for personalization of the news feed or other Mi
 Microsoft Edge WebDriver allows developers to drive the Microsoft Edge browser using the [WebDriver protocol](https://www.w3.org/TR/webdriver2/).  The Microsoft Edge WebDriver is an executable file `msedgedriver.exe` separate from Microsoft Edge. Developers can call the driver from  client code, such as a test script.  By default, Microsoft Edge WebDriver sends diagnostic data such as the status of the [New Session WebDriver command](https://www.w3.org/TR/webdriver2/#new-session) to Microsoft.  To turn off the diagnostic data collection for Microsoft Edge WebDriver, set the `MSEDGEDRIVER_TELEMETRY_OPTOUT` environment variable to `1`.  For more information about Microsoft Edge WebDriver, see [Use WebDriver to automate Microsoft Edge automation](/microsoft-edge/webdriver-chromium).
 
 
+<!-- ====================================================================== -->
+## Rewards
```

Comment:
@majenny move Rewards section to the 'R' section instead of between 'M' and 'N' section - or change heading to "Microsoft Rewards"

gh rendered section:
https://github.com/MicrosoftDocs/edge-developer/blob/user/majenny/ComposePrivacy/microsoft-edge/privacy-whitepaper/index.md#rewards